.travis.yml

```diff
@@ -12,7 +12,12 @@ matrix:
 before_install:
   - if [ $TRAVIS_OS_NAME = linux ]; then sudo apt-get install git gnupg2 clang-format-3.9; fi
   - if [ $TRAVIS_OS_NAME = osx ]; then brew install git gnupg clang-format || true; fi
-  - if [ $TRAVIS_OS_NAME = windows ]; then choco install --yes --force --debug make gpg4win-lite || true; fi
+  - | 
+      if [ $TRAVIS_OS_NAME = windows ]; then 
+      choco install --yes --force --debug make gnupg || true; 
+      rm /usr/bin/gpg
+      diskperf -y
```

Comment:
cygwin ships with an outdated version of gpg which is incompatible with gopass (it tries to migrate the gpg keystore).
`diskperf -y` must be called on windows to suppress a warning in GPG stdout.